src/Http/Http.Extensions/test/RequestDelegateGenerator/Baselines/RequestDelegateValidateGeneratedFormCode.generated.txt

```diff
@@ -487,4 +487,26 @@ namespace Microsoft.AspNetCore.Http.Generated
         private static readonly Action<ILogger, string, string, Exception?> _invalidFormRequestBody =
             LoggerMessage.Define<string, string>(LogLevel.Debug, new EventId(8, "InvalidFormRequestBody"), "Failed to read parameter \"{ParameterType} {ParameterName}\" from the request body as form.");
     }
+
+    file static class GeneratedMetadataConstants
+    {
+        public static readonly string[] JsonContentType = new [] { "application/json" };
+        public static readonly string[] PlaintextContentType = new [] { "text/plain" };
+    }
+
+    file sealed class GeneratedProducesResponseTypeMetadata : IProducesResponseTypeMetadata
```

Comment:
This code is not used in the generated source. Can we conditionally add this code only if it is necessary/used?

Reply:
Fixed this. Ended up introducing the concept of `helperTypes` in addition to `helperMethods` - the technique for each is the same but obviously the types need to be dynamically injected at a different spot to some of the methods.